cmd/benchmark/main.go

```diff
@@ -203,9 +217,22 @@ func getTotalBalance(insSDK *sdk.SDK, members []*sdk.Member) (totalBalance *big.
 
 	// execute all queries in parallel
 	for i := 0; i < nmembers; i++ {
-		go func(m *sdk.Member, num int) {
+		go func(m sdk.Member, num int) {
 			res := Result{num: num}
-			res.balance, res.err = insSDK.GetBalance(m)
+			balance, deposits, err := insSDK.GetBalance(m)
+			for _, d := range deposits {
+				depositBalanceStr, ok := d.(map[string]interface{})["balance"].(string)
+				if !ok {
+					err = errors.New("failed to get balance from deposit")
```

Comment:
In case of this error you just continue, is it what you want?

Reply:
This is correct. But also need to handle previous err if it is not nil.
Done